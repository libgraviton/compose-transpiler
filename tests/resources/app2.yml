header:
  template: customHeader
components:
  fred:
    mixins:
      more:
    additions:
      literalDude: [1, 2]

