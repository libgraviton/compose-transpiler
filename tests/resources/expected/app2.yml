version: 1
services:
  fred:
    image: 'fred:latest'
    addedKey: yeah
    literalDude: [1, 2]
networks:
  internal:
    external: false
